.github/workflows/jest-testing.yml

```diff
@@ -0,0 +1,37 @@
+name: Run Jest testing suite
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - development
+    paths-ignore:
+      - "**/total-rewards.txt"
+  pull_request:
+
+env:
+  NODE_ENV: "test"
+
+jobs:
+  testing:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
```

Comment:
```suggestion
          node-version: '20.10.0'
```

Always use `20.10.0` LTS across all of our codebases.

Reply:
No problem, just that any `20.x` version is an LTS version, latest being `20.11.1`, by not using the last we skip the patches and bug fixes so I thought I would be nicer to use the latest one
![image](https://github.com/ubiquity/ts-template/assets/9807008/fbf0792e-4019-42f4-bfb8-074a2486c4c9)

Reply:
x defaults to LTS? Didn't know this. Seems like a good idea but at the same time wouldn't defining 20.10.0 across the board simplify things? Even for @types/node we should set 20.10.0 for example.